packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -24,6 +22,60 @@ export class UsaBanner extends React.PureComponent {
     const t =
       this.props.locale === 'es' ? SpanishTranslations.usaBanner : EnglishTranslations.usaBanner;
 
+    // TODO: Find out why browser (Chrome and Safari) does not show role="img"
+    // when referencing UsaFlag inline svg file
+    const IconUsaFlag = () => (
+      <svg
+        role="img"
+        width="16"
+        height="11"
+        xmlns="http://www.w3.org/2000/svg"
+        className="ds-c-usa-banner__header-flag"
+        // eslint-disable-next-line no-template-curly-in-string
+        aria-labelledby={`banner-flag-title-${this.id} banner-flag-description-${this.id}`}
+        focusable="false"
+      >
+        {/* eslint-disable-next-line no-template-curly-in-string */}
+        <title id={`banner-flag-title-${this.id}`}>Flag</title>
+        {/* eslint-disable-next-line no-template-curly-in-string */}
+        <desc id={`banner-flag-description-${this.id}`}>U.S. flag</desc>
+        <g fill="none" fillRule="evenodd">
+          <path fill="#FFF" d="M0 0h16v11H0z" />
+          <path fill="#DB3E1F" d="M8 0h8v1H8z" />
+          <path fill="#1E33B1" d="M0 0h8v7H0z" />
+          <path fill="#DB3E1F" d="M8 2h8v1H8zM8 4h8v1H8zM8 6h8v1H8zM0 8h16v1H0zM0 10h16v1H0z" />
+          <path
+            fill="#FFF"
+            d="M1 1h1v1H1zM2 3h1v1H2zM1 5h1v1H1zM3 1h1v1H3zM4 3h1v1H4zM3 5h1v1H3zM5 1h1v1H5zM6 3h1v1H6zM5 5h1v1H5z"
+          />
+        </g>
+      </svg>
+    );
+
+    // TODO: Find out why the browser (Chrome and Safari) does not show role="img"
+    // when referencing lock inline svg file
+    const IconLock = () => (
+      <svg
+        role="img"
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 52 64"
+        className="ds-c-usa-banner__lock-image"
+        // eslint-disable-next-line no-template-curly-in-string
+        aria-labelledby={`banner-lock-title-${this.id} banner-lock-description-${this.id}`}
+        focusable="false"
+      >
+        {/* eslint-disable-next-line no-template-curly-in-string */}
+        <title id={`banner-lock-title-${this.id}`}>Lock</title>
+        {/* eslint-disable-next-line no-template-curly-in-string */}
+        <desc id={`banner-lock-description-${this.id}`}>A locked padlock</desc>
+        <path
+          fill="#000000"
+          fillRule="evenodd"
+          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"
+        />
+      </svg>
+    );
+
```

Comment:
Svg files for `U.S flag and lock` are placed here to fix the accessibility issue of browsers not reading the `role="img"` flag even after updating the script to pass `role, aria-labelledby, title and desc` svgmin options to the `dist/imag/` folder. 
Also noted that uswds banner have the `lock svg` file function in the code instead of on the external image file as well - is it to workaround the accessibility issue of screen reader not reading correctly?

Reply:
I would prefer to leave the SVG usage as it originally was, even if the title, description and role are not working. we can always fix that later by filing a ticket to fix svg min

Reply:
the problem here is that our SVG in react system is to import the SVG file directly, and use it inline. This promotes code reuse and consistently with our icons, and is better than writing the SVG inline manually. also, regardless of svgmin, you should still be able to use role, just apply it to the imported SVG as another prop

Reply:
Ah, I didn't think of applying the role to the imported svg as another prop. Will give that a try to see if that fixes the issue of screen reader not recognizing the `U.S. flag` and `Lock` SVG file as an image like when the svg inline is added manually.

Reply:
Updated to use the inline SVG and pass in `role` prop

Reply:
you dont need these variables anymore, you should just be using the icon components directly in the JSX